### DISCUSSION TITLE
Mi primer web site

### DISCUSSION THREAD

**Discussion by stvnmc** [Launch-X-Latam/MisionFrontEnd#280] [General]
https://github.com/stvnmc/Cat-box-

Me gustaria leer opiniones.

**Comment by OscarJG20** [comment]
1. Los estilos del sitio comienzan a verse bien.
2. Yo le pondría nombres más descriptivos a las imágenes (por cuestiones de SEO).
3. Y checa tu contenido desbordado, a la larga eso te dará más problemas si piensas hacerlo responsive.

![image](https://user-images.githubusercontent.com/64516675/198894299-b125e825-2b01-4bab-8baa-6d70518c05f7.png)
